lib/wine_o_inventory_api_web/endpoint.ex

```diff
@@ -18,9 +18,15 @@ defmodule WineOInventoryApiWeb.Endpoint do
   # when deploying your static files in production.
   plug Plug.Static,
     at: "/",
-    from: :wine_o_inventory_api,
+    from: {:wine_o_inventory_api, "priv/static"},
     gzip: false,
-    only: ~w(assets fonts images favicon.ico robots.txt)
+    only:
+      ~w(assets fonts images favicon.ico robots.txt asset-manifest.json manifest.json logo192.png logo512.png)
+
+  plug Plug.Static,
+    at: "/static",
+    from: {:wine_o_inventory_api, "priv/static/static"},
```

Comment:
What is going on here? why `static/static`? Could we name the directory to be a bit more expressive? I've seen examples online with something like `webapp`